src/kOS/Suffixed/AggregateResourceValue.cs

```diff
@@ -1,44 +1,41 @@
 ﻿using System.Collections.Generic;
 using kOS.Safe.Encapsulation;
+using kOS.Safe.Encapsulation.Suffixes;
+using kOS.Suffixed.Part;
 
 namespace kOS.Suffixed
 {
     public class AggregateResourceValue : Structure
     {
         private readonly string name;
+        private readonly SharedObjects shared;
         private double amount;
         private double capacity;
-        private readonly ListValue parts;
+        private readonly SuffixedList<PartValue> parts;
```

Comment:
Now it's my turn to turn the tables on you and ask you to defend redundancy.

What was wrong with ListValue<PartValue>?  What was the incentive for making a special kind of derivative of ListValue which is only templated for suffixable objects as its elements, when you still have to go and be more specific anyway and supply the exact type of suffixable object (in this case PartValue) anyway like this?